src/views/perms/AssetPermission/components/AccountFormatter.vue

```diff
@@ -185,6 +193,18 @@ export default {
   computed: {
     virtualAccount() {
       return virtualAccount
+    },
+    iRealChoices: {
+      get() {
+        let choices = this.realChoices.slice()
+        if (!this.enableNoneAccount) {
+          choices = choices.filter(i => i.value !== NoneAccount)
+        }
+        if (!this.enableExcludeAccounts) {
+          choices = choices.filter(i => i.value !== ExcludeAccount)
+        }
+        return choices
+      }
     }
   },
   watch: {
```

Comment:
Unfortunately, I do not have access to the provided code snippet you mentioned. Please share the code and describe its purpose so I can provide more specific guidance on whether it is relevant to my capabilities currently, as well as any knowledge that existed up until 2021-09-01.